lib/ansible/modules/cloud/azure/azure_rm_virtualmachineextension.py

```diff
@@ -211,6 +211,9 @@ def exec_module(self, **kwargs):
         for key in list(self.module_arg_spec.keys()):
             setattr(self, key, kwargs[key])
 
+        if self.module._name == 'azure_rm_virtualmachine_extension':
+            self.module.deprecate("The 'azure_rm_virtualmachine_extension' module has been renamed to 'azure_rm_virtualmachineextension'", version='2.12')
```

Comment:
what does version=2.12 mean?

Reply:
that it will be deprecated in Ansible 2.12